pandas-stubs/core/arrays/sparse/dtype.pyi

```diff
@@ -1,33 +1,22 @@
+import numpy as np
+
 from pandas._typing import (
     Dtype,
     Scalar,
+    npt,
 )
 
 from pandas.core.dtypes.base import ExtensionDtype
 from pandas.core.dtypes.dtypes import (
     register_extension_dtype as register_extension_dtype,
 )
 
-# merged types from pylance
-
 class SparseDtype(ExtensionDtype):
-    def __init__(self, dtype: Dtype = ..., fill_value: Scalar | None = ...) -> None: ...
-    def __hash__(self): ...
-    def __eq__(self, other) -> bool: ...
-    @property
-    def fill_value(self): ...
-    @property
-    def kind(self): ...
-    @property
-    def type(self): ...
+    def __init__(
+        self, dtype: Dtype | npt.DTypeLike = ..., fill_value: Scalar | None = ...
+    ) -> None: ...
     @property
-    def subtype(self): ...
+    def fill_value(self) -> Scalar | None: ...
     @property
-    def name(self): ...
-    @classmethod
-    def construct_array_type(cls): ...
-    @classmethod
-    def construct_from_string(cls, string): ...
-    @classmethod
-    def is_dtype(cls, dtype): ...
-    def update_dtype(self, dtype): ...
+    def subtype(self) -> Dtype: ...
+    def update_dtype(self, dtype: SparseDtype | npt.DTypeLike) -> SparseDtype: ...
```

Comment:
I don't think `subtype` and `update_dtype` are documented, so let's not include them here

Reply:
Done.